vms/platformvm/block/executor/helpers_test.go

```diff
@@ -59,6 +59,12 @@ const (
 
 	defaultWeight = 10000
 	trackChecksum = false
+
+	apricotPhase3 fork = iota
+	apricotPhase5
+	banffFork
+	cortinaFork
+	durangoFork
```

Comment:
style nit: the "Fork" suffix should be consistently used or dropped

Reply:
Dropped